portfolio/src/main/java/com/google/sps/servlets/LoginStatusServlet.java

```diff
@@ -0,0 +1,22 @@
+import com.google.appengine.api.users.UserService;
```

Comment:
Two lint errors in this file: the import in line 7 is out of order, and there is no newline at end of file.